package.json

```diff
@@ -3,7 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
```

Comment:
Are you using these 3 packages? If not remove them to avoid having unnecessary dependencies